airbyte-ci/connectors/metadata_service/orchestrator/orchestrator/logging/publish_connector_lifecycle.py

```diff
@@ -56,19 +57,45 @@ def stage_to_log_level(stage_status: StageStatus) -> str:
         else:
             return "info"
 
+    def _commit_link(commit_sha: str) -> str:
+        """Create a markdown link to a commit."""
+        commit_url = f"{REPO_URL}/commit/{commit_sha}"
+        return f"\ncommit: <{commit_url}|{commit_sha}>"
+
+    def _user_mention(user_identifier: str) -> str:
+        """Create a markdown link to a user."""
+        return f"\nauthor: {user_identifier}"
+
     @staticmethod
     def create_log_message(
         lifecycle_stage: PublishConnectorLifecycleStage,
         stage_status: StageStatus,
         message: str,
+        commit_sha: str = None,
+        user_identifier: str = None,
     ) -> str:
         emoji = stage_status.to_emoji()
-        return f"*{emoji} _{lifecycle_stage}_ {stage_status}*: {message}"
+        final_message = f"*{emoji} _{lifecycle_stage}_ {stage_status}*:\n{message}"
+
+        if user_identifier:
+            final_message += PublishConnectorLifecycle._user_mention(user_identifier)
+
+        if commit_sha:
+            final_message += PublishConnectorLifecycle._commit_link(commit_sha)
+
+        return final_message
 
     @staticmethod
-    def log(context: OpExecutionContext, lifecycle_stage: PublishConnectorLifecycleStage, stage_status: StageStatus, message: str):
+    def log(
+        context: OpExecutionContext,
+        lifecycle_stage: PublishConnectorLifecycleStage,
+        stage_status: StageStatus,
+        message: str,
+        commit_sha: str = None,
+        user_identifier: str = None,
+    ):
         """Publish a connector notification log to logger and slack (if enabled)."""
-        message = PublishConnectorLifecycle.create_log_message(lifecycle_stage, stage_status, message)
+        message = PublishConnectorLifecycle.create_log_message(lifecycle_stage, stage_status, message, commit_sha, user_identifier)
```

Comment:
Can the slack bot also invite the committer to the `#connector-publish-updates` channel if they are not channel member yet?

Reply:
Let me look into that! Though it also may be useful to let users opt out of this feature 😅